### PR TITLE
AAP-65871: Fix openapi spec for oauth2 algorithms

### PR DESCRIPTION
**Pull request by bhavenst**
Removes double blank value for algorithm
  - blank already defined in ALGORITHM_TYPES

## Description

Just a small update to fix up OAuth2Application openapi schema (double blank value in enum list).  Should have no other effect.

## Type of Change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Test update
- [ ] Refactoring (no functional changes)
- [x] Development environment change
- [ ] Configuration change

## Self-Review Checklist
- [x] I have performed a self-review of my code
- [x] I have added relevant comments to complex code sections
- [x] I have updated documentation where needed
- [x] I have considered the security impact of these changes
- [x] I have considered performance implications
- [x] I have thought about error handling and edge cases
- [x] I have tested the changes in my local environment

### Steps to Test
1. Generate gateway openapi spec.
2. See that oauth2application algorithm enum list does not contain double blank values
3. 

### Expected Results
openapi spec is correct

## Release Notes

* **Bug Fixes**
  * The algorithm field for OAuth2 applications is now required and cannot be left blank, ensuring proper configuration validation.

### CODE REVIEW

File: ansible_base/oauth2_provider/serializers/application.py [hunk 1]
```diff
@@ -24,6 +24,7 @@ class Meta:
             'redirect_uris': {'label': _('Redirect URIs')},
             'app_url': {'label': _('Application URL')},
             'skip_authorization': {'label': _('Skip Authorization')},
+            'algorithm': {'allow_blank': False},
```
Comment (john-westcott-iv) [inline ansible_base/oauth2_provider/serializers/application.py]: Do you want to do an uppercase 'A' ''abel?

**Review by john-westcott-iv — APPROVED** [review]

File: ansible_base/oauth2_provider/serializers/application.py (hunk 1 above)
Comment (bhavenst) [reply ansible_base/oauth2_provider/serializers/application.py]: Just added label with A

File: ansible_base/oauth2_provider/serializers/application.py [hunk 2]
```diff
@@ -24,6 +24,7 @@ class Meta:
             'redirect_uris': {'label': _('Redirect URIs')},
             'app_url': {'label': _('Application URL')},
             'skip_authorization': {'label': _('Skip Authorization')},
+            'algorithm': {'allow_blank': False, 'label': _('Algorithm')},
```
[the remark this answers is not in the document]
Comment (bhavenst) [reply ansible_base/oauth2_provider/serializers/application.py:27]: This is not true.  blank is a valid explicit entry in the enum and works fine.
